yucca/umbrel-app.yml

```diff
@@ -0,0 +1,29 @@
+manifestVersion: 1.1
+id: yucca
+category: automation
+name: Yucca
+version: "0.10.0"
+tagline: Simple NVR solution video surveillance
+description: >-
+  Simple NVR solution video surveillance for home and business.  
+  Perfect for both a solution for safe cities with thousands of cameras and launching a subscriber service, as well as for home use self-hosted.
+
+releaseNotes: >-
+  Init Yucca on Umbrel
+
+developer:  Yucca Stream
+website: https://yucca.app/
+repo: ""
+support: https://docs.yucca.app/en/support/
+port: 9910
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+  - 4.jpg
+  - 5.jpg
+path: ""
+deterministicPassword: false
+dependencies: []
+submitter: jidckii
```

Comment:
Since you are a developer of YuccaStream, you can change this to:
`submitter: YuccaStream`

The reason for this is so that users who go to download the app can easily see that it is from the official developers of the app